cocos/spine/skeleton.ts

```diff
@@ -158,81 +152,93 @@ export class SpineSocket {
 
 js.setClassAlias(SpineSocket, 'sp.Skeleton.SpineSocket');
 
-/**
- * @en
- * The skeleton of Spine <br/>
- * <br/>
- * (Skeleton has a reference to a SkeletonData and stores the state for skeleton instance,
- * which consists of the current pose's bone SRT, slot colors, and which slot attachments are visible. <br/>
- * Multiple skeletons can use the same SkeletonData which includes all animations, skins, and attachments.) <br/>
- * Cocos Creator supports spine versions lower than 3.8.99.
- * @zh
- * Spine 骨骼动画 <br/>
- * <br/>
- * (Skeleton 具有对骨骼数据的引用并且存储了骨骼实例的状态，
- * 它由当前的骨骼动作，slot 颜色，和可见的 slot attachments 组成。<br/>
- * 多个 Skeleton 可以使用相同的骨骼数据，其中包括所有的动画，皮肤和 attachments。
- * Cocos Creator 支持 spine 版本最高到3.8.99。
- * @class Skeleton
- * @extends UIRenderer
- */
 @ccclass('sp.Skeleton')
-@help('i18n:cc.Spine')
+@help('i18n:sp.Skeleton')
 @menu('Spine/Skeleton')
 @executeInEditMode
 export class Skeleton extends UIRenderer {
     public static SpineSocket = SpineSocket;
     public static AnimationCacheMode = AnimationCacheMode;
 
+    @serializable
+    protected _skeletonData: SkeletonData | null = null;
+    @serializable
+    protected defaultSkin = '';
+    @serializable
+    protected defaultAnimation = '';
     /**
-     * @internal Since v3.7.2, this is an engine private interface.
+     * @en Indicates whether to enable premultiplied alpha.
+     * You should disable this option when image's transparent area appears to have opaque pixels,
+     * or enable this option when image's half transparent area appears to be darken.
+     * @zh 是否启用贴图预乘。
+     * 当图片的透明区域出现色块时需要关闭该选项，当图片的半透明区域颜色变黑时需要启用该选项。
      */
-    get drawList () { return this._drawList; }
+    @serializable
+    protected _premultipliedAlpha = true;
+    @serializable
+    protected _timeScale = 1;
+    @serializable
+    protected _cacheMode = AnimationCacheMode.REALTIME;
+    @serializable
+    protected _defaultCacheMode: AnimationCacheMode = AnimationCacheMode.REALTIME;
+    @serializable
+    protected _sockets: SpineSocket[] = [];
+    @serializable
+    protected _useTint = false;
+    @serializable
+    protected _debugMesh = false;
+    @serializable
+    protected _debugBones = false;
+    @serializable
+    protected _debugSlots = false;
+    @serializable
+    protected _enableBatch = false;
 
-    protected _updateBuiltinMaterial (): Material {
-        const material = builtinResMgr.get<Material>('default-spine-material');
-        return material;
-    }
+    protected _runtimeData: spine.SkeletonData | null = null;
+    public _skeleton: spine.Skeleton = null!;
```

Comment:
Did we export this interface before?

Reply:
_skeleton always is public

Reply:
Yeah, for now we need to keep it that way